script/upgrades/MU01-mcmint/governance/MU01-Baklava-CGP.sol

```diff
@@ -15,28 +15,218 @@ import { IRegistry } from "mento-core/contracts/common/interfaces/IRegistry.sol"
+      isMedianDeltaBreakerEnabled: true,
+      medianDeltaBreakerThreshold: FixidityLib.newFixedFraction(3, 100), // 0.03
+      medianDeltaBreakerCooldown: 30 minutes,
+      isValueDeltaBreakerEnabled: false,
+      valueDeltaBreakerThreshold: FixidityLib.wrap(0),
+      valueDeltaBreakerReferenceValue: 0,
+      valueDeltaBreakerCooldown: 0,
+      referenceRateFeedID: cUSD,
+      asset0_timeStep0: 5 minutes,
+      asset0_timeStep1: 1 days,
+      asset0_limit0: 1_000_000,
+      asset0_limit1: 5_000_000,
+      asset0_limitGlobal: 0,
+      asset0_flags: L0 | L1
+    });
+    poolConfigs.push(cUSDCeloConfig);
+
+    // Set the exchange ID for the reference rate feed
+    referenceRateFeedIDToExchangeId[cUSDCeloConfig.referenceRateFeedID] = getExchangeId(
+      cUSDCeloConfig.asset0,
+      cUSDCeloConfig.asset1,
+      cUSDCeloConfig.isConstantSum
+    );
+
+    // Create pool configuration for cEUR/CELO pool
+    cEURCeloConfig = PoolConfiguration({
+      asset0: cEUR,
+      asset1: celo,
+      isConstantSum: false,
+      spread: FixidityLib.newFixedFraction(25, 10000), // 0.0025
+      referenceRateResetFrequency: 5 minutes,
+      minimumReports: 5,
+      stablePoolResetSize: 1_800_000 * 1e18, // 1.8mil
+      isMedianDeltaBreakerEnabled: true,
+      medianDeltaBreakerThreshold: FixidityLib.newFixedFraction(3, 100), // 0.03
+      medianDeltaBreakerCooldown: 30 minutes,
+      isValueDeltaBreakerEnabled: false,
+      valueDeltaBreakerThreshold: FixidityLib.wrap(0),
+      valueDeltaBreakerReferenceValue: 0,
+      valueDeltaBreakerCooldown: 0,
+      referenceRateFeedID: cEUR,
+      asset0_timeStep0: 5 minutes,
+      asset0_timeStep1: 1 days,
+      asset0_limit0: 1_000_000,
+      asset0_limit1: 5_000_000,
+      asset0_limitGlobal: 0,
+      asset0_flags: L0 | L1
+    });
+    poolConfigs.push(cEURCeloConfig);
+
+    // Set the exchange ID for the reference rate feed
+    referenceRateFeedIDToExchangeId[cEURCeloConfig.referenceRateFeedID] = getExchangeId(
+      cEURCeloConfig.asset0,
+      cEURCeloConfig.asset1,
+      cEURCeloConfig.isConstantSum
+    );
+
+    // Create pool configuration for cBRL/CELO pool
+    cBRLCeloConfig = PoolConfiguration({
+      asset0: cBRL,
+      asset1: celo,
+      isConstantSum: false,
+      spread: FixidityLib.newFixedFraction(25, 10000), // 0.0025
+      referenceRateResetFrequency: 5 minutes,
+      minimumReports: 5,
+      stablePoolResetSize: 3_000_000 * 1e18, // 3min
+      isMedianDeltaBreakerEnabled: true,
+      medianDeltaBreakerThreshold: FixidityLib.newFixedFraction(3, 100), // 0.03
+      medianDeltaBreakerCooldown: 30 minutes,
+      isValueDeltaBreakerEnabled: false,
+      valueDeltaBreakerThreshold: FixidityLib.wrap(0),
+      valueDeltaBreakerReferenceValue: 0,
+      valueDeltaBreakerCooldown: 0,
+      referenceRateFeedID: cBRL,
+      asset0_timeStep0: 5 minutes,
+      asset0_timeStep1: 1 days,
+      asset0_limit0: int48(1_000_000),
```

Comment:
Sorry, for picking on conventions again...Is there a reason why the cBRL limits are typed?

Reply:
I did this to solve some implicit type conversion bug. Maybe there's a cleaner solution